packages/starknet-snap/test/src/getErc20TokenBalance.test.ts

```diff
@@ -35,9 +35,36 @@ describe('Test function: getErc20TokenBalance', function () {
     sandbox.restore();
   });
 
-  it('should get the ERC-20 token balance correctly', async function () {
+  it('should get the ERC-20 token balance correctly from pending if account is deployed', async function () {
```

Comment:
the test case should be
- should get the ERC-20 token balance correctly
- should get ERC-20 token balance with BlockIdentifier 'pending' if the account is deployed 
- should get ERC-20 token balance with BlockIdentifier 'latest' if the account is not deployed